SEM/datamodels/plugins/thermofischer.py

```diff
@@ -0,0 +1,86 @@
+from pathlib import Path
+import dlite
+
+import configparser
+from PIL import Image
+from PIL.TiffTags import TAGS
+
+# Add new tags to TIFF
+TAGS[34682] = "ThermoFischer"
+
+def parse_ini(ini_str : str):
+    """
+    Parse a INI file from string to dict.
+    """
+    config = configparser.ConfigParser()
+    config.read_string(ini_str)
+
+    # Convert to dict
+    metadata = dict()
+    sections = config.sections()
+    for section in sections:
+        items = config.items(section)
+        metadata[section] = dict(items)
+
+    return metadata
```

Comment:
Same comment as for hitachi.py:
- make `parse_ini()` a static method
- it should also be safe to make `TAGS[34682]` an attribute

Reply:
I have now moved TAGS[34682] = "ThermoFisher" into the open() function and made parse_ini a static method.